### PR TITLE
Adding new Device

### PR DESCRIPTION
**Pull request by GiorgosM1995**
#### Summary

...

#### Changes

- ...
- ...

#### Checklist for Reviewers

- [x] Title and description should be descriptive (Not just a serial number for example).
- [x] `profileIDs` should not be `vendorID` and should be a unique value for every profile.
- [x] All devices should be listed in the vendor's `index.yaml` file.
- [x] Firmware versions can not be changed.
- [x] At least 1 image per device and should be transparent.

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/cicicom/s-lg-i3.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Sensors that this device features (optional)
+# Valid values are:
+# 4-20 ma, accelerometer, altitude, analog input, auxiliary, barometer, battery, button, bvoc, co, co2, conductivity,
+# current, digital input, dissolved oxygen, distance, dust, energy, gps, gyroscope, h2s, humidity, iaq, level, light,
+# lightning, link, magnetometer, moisture, motion, no, no2, o3, particulate matter, ph, pir, pm2.5, pm10, potentiometer,
+# power, precipitation, pressure, proximity, pulse count, pulse frequency, radar, rainfall, rssi, smart valve, snr, so2,
+# solar radiation, sound, strain, surface temperature, temperature, tilt, time, tvoc, uv, vapor pressure, velocity,
+# vibration, voltage, water potential, water, weight, wifi ssid, wind direction, wind speed.
+sensors:
+   - radar, magnetometer
```
Comment (LDannijs) [inline vendor/cicicom/s-lg-i3.yaml:33]: Thank you for creating a PR

Please run `make validate fmt` in a command prompt to adhere to our schema. This for example needs to become:
```suggestion
   - radar
   - magnetometer
```

**Comment by LDannijs** [comment]
You've not yet ran `make validate fmt`, i still see some issues. Please make sure to read our README on how to do it: https://github.com/TheThingsNetwork/lorawan-devices?tab=readme-ov-file#prerequisites

**Comment by LDannijs** [comment]
Please run `make validate fmt` until there are no more errors that show up.

**Comment by LDannijs** [comment]
Thank you! My colleague @Jaime-Trinidad will merge it when he looks at it.

**Comment by GiorgosM1995** [comment]
Thanks for the info


Best Regards



Georgios Mouratidis

IoT Engineer

[cid:6c547002-5158-4401-a41b-1c1db0675efe]

Kosta Varnali 12

15233 Chalandri, Athens, Greece

Tel: +30 210 6391072 (Ext. 650)

Email: ***@***.******@***.***> | Web: www.cicicom.gr<http://www.cicicom.gr/>



[cid:60d15dd6-3a49-4e33-9a57-e3979e0be080]

This e-mail (including any attached documents) is proprietary and confidential and may contain legally privileged information. It is intended for the named recipient(s) only. If you are not the intended recipient, you may not review, retain, copy or distribute this message, and we kindly ask you to notify the sender by reply e-mail immediately and delete this message from your system. Thank you.

E-mail is susceptible to unauthorized alterations which may corrupt the message of the e-mail. Our Company only sends and receives e-mails on the basis that is not liable for any unauthorized alterations made to the e-mails.

________________________________
Από: Lars Dannijs ***@***.***>
Στάλθηκε: Τρίτη, 21 Μαΐου 2024 2:15 μμ
Προς: TheThingsNetwork/lorawan-devices ***@***.***>
Κοιν.: George Mouratidis ***@***.***>; Author ***@***.***>
Θέμα: Re: [TheThingsNetwork/lorawan-devices] Adding new Device (PR #791)


Thank you! My colleague @Jaime-Trinidad<https://github.com/Jaime-Trinidad> will merge it when he looks at it.

**Review by Jaime-Trinidad — APPROVED** [review]

File: vendor/cicicom/s-lg-i3.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    numeric: 14
+    hardwareVersions:
+      - '1.0'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: s-lg-i3
```
Comment (LDannijs) [inline vendor/cicicom/s-lg-i3.yaml:21]: the id for the profile should point to the filename of the profile
```suggestion
        id: s-lg-i3-profile
```

**Comment by LDannijs** [comment]
I am seeing some prettier issues still. Please make sure you have read our README and make sure you run `make validate fmt` every time you want to push changes.

**Comment by GiorgosM1995** [comment]
I cannot run make validate fmt gives me this error :
-z was unexpected at this time.
make: *** [Makefile:41: validate] Error 255

**Comment by LDannijs** [comment]
Have you made sure you have installed:
- Node.js version 16.x
(This is recommended, as newer versions may give errors)
- npm version 8.x
- Go version 1.18.x

and ran `make deps`? https://github.com/TheThingsNetwork/lorawan-devices?tab=readme-ov-file#prerequisites

**Comment by GiorgosM1995** [comment]
I had a newer version of Node i am installing the one you said me and i am trying again

**Comment by GiorgosM1995** [comment]
make deps give me this error :
pushd ./tools/runscript && go install . && popd
The syntax of the command is incorrect.
make: *** [Makefile:31: deps] Error 1

Also make validate fmt:
-z was unexpected at this time.
make: *** [Makefile:41: validate] Error 255mak

**Comment by LDannijs** [comment]
May i ask what operating system you are using?

**Comment by GiorgosM1995** [comment]
Edition: Windows 10 Home
Version: 22h2
OS build: 19045.4412

**Comment by LDannijs** [comment]
As i suspected. Have you installed Windows Subsystem for Linux (WSL)? If not, install that or install linux (on a VM is possible)

**Comment by LDannijs** [comment]
Have you made sure you are using node 16? try `node -v`

**Comment by LDannijs** [comment]
Everything looks good now, thanks! My colleague will merge it when he gets to it.

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**